src/client/src/services/openFin/epics/analyticsServiceEpic.ts

```diff
@@ -0,0 +1,16 @@
+import { Action } from 'redux'
+import { ofType } from 'redux-observable'
+import { ignoreElements, tap } from 'rxjs/operators'
+import { ApplicationEpic } from '../../../ApplicationEpic'
+import { ACTION_TYPES as ANALYTICS_ACTION_TYPES, AnalyticsActions } from '../../../ui/analytics'
+
+type FetchAnalyticsAction = ReturnType<typeof AnalyticsActions.fetchAnalytics>
+
+export const connectAnalyticsServiceToOpenFinEpic: ApplicationEpic = (action$, state$, { openFin }) =>
+  action$.pipe(
+    ofType<Action, FetchAnalyticsAction>(ANALYTICS_ACTION_TYPES.ANALYTICS_SERVICE),
+    tap<FetchAnalyticsAction>((action: FetchAnalyticsAction) =>
```

Comment:
Eventually we should just use a subscribe. Doing things like this in tap is a bit crap and also hard to test. For now this will do as it is a step in the right direction.